web/blog/2022-05-31-filip-intro.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Meet the team - Filip Sodić, Founding Engineer
+authors: [matijasos]
+image: /img/shayne-intro-banner.png
+tags: [wasp, new-hire]
+---
+
+import Link from '@docusaurus/Link';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import InBlogCta from './components/InBlogCta';
+import WaspIntro from './_wasp-intro.md';
+
+![Welcome Filip!](../static/img/shayne-intro-banner.png)
+
+Find Filip on [GitHub](https://github.com/sodic).
+
+<!--truncate-->
+
+<WaspIntro />
+<InBlogCta />
+
+#### Let’s start with two truths and a lie about yourself.
+
+- I found out about Wasp through a friend/former colleague and still haven’t
+  bought him a drink to say thanks.
+- I’ve created my own Turing-complete functional programming language that
+  compiles to JavaScript.
+- I wasted nearly 30 hours customizing my setup to maximize productivity and
+  probably ended up in a less productive state
+
+#### Which statement was the lie above? Any interesting stories to share?
+
+The last one was a lie. I wish it were only 30 hours :)
+
+I did create my own language, but it’s still far from being practical. For one,
+it can’t print anything to the screen (or perform any side effects).
+
+I never did buy that drink. He’s not letting me forget about it, though.
+
+#### Why did you join Wasp? What did you do before?
+
+I've wanted a job where I'd get a chance to design a language and build a
+compiler for quite some time now. While I did develop my own language on the
+side (and even graduated with it), I thought getting paid for doing something
+similar this early in my career was pretty far-fetched.
+
+Almost all my paid work had something to do with the web, whether on the
+client-side or the server-side. I enjoy nearly all parts of web development
+(yes, even styling websites with CSS :), but one thing I've always found
+annoying was dealing with all the boilerplate. Whenever I start building
+something, I spend most of my time implementing the same repetitive stuff
+(e.g., authentication, databases, SPA routing, API routes, etc.) and relearning
+all the best practices that come with it.
+
+And then, one day, a friend of mine mentioned Wasp - a DSL aiming to eliminate
+boilerplate from web apps. The product was a perfect fit based on everything I
+know and want to do! Also, the founders (Martin and Matija) and I shared an
+advisor in college, and I've only heard great things about them.
+
+The whole thing sounded almost too good to be true, so joining the company was
```

Comment:
```suggestion
The whole thing sounded almost too good to be true, so joining them in working on Wasp was
```

Reply:
I think "joining them to work on Wasp" is correct, but am not sure. I'll see what Grammarly says.

Reply:
No help there unfortunately, let's ask our dedicated native speaker @shayneczyzewski.

Reply:
I think both are probably valid, but I'd prefer "joining them to work on Wasp" over "joining them in working on Wasp". The latter has a slightly overly formal tone to me.  🤷🏻‍♂️